precompiles/ibc/IBC.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+address constant BANK_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000001008;
```

Comment:
Could we udpate this to 0x0000000000000000000000000000000000001009 ? Oracle is about to be merged https://github.com/sei-protocol/sei-chain/pull/1445

Reply:
Updated